rpc/wrtc_signaling_answerer.go

```diff
@@ -247,16 +267,18 @@ func (ans *webrtcSignalingAnswerer) answer(client webrtcpb.SignalingService_Answ
 		}
 	}()
 
+	receivedInitRequest = false
```

Comment:
The whole patch is really just this and the next 2 lines. Only `client.Recv` erroring should result in a "return false" and every other return case should have "receivedInitRequest = true".